TauAnalysis/MCEmbeddingTools/test/BuildFile.xml

```diff
@@ -1,4 +0,0 @@
-<bin name="testTauEmbeddingProducers" file="TestDriver.cpp">
```

Comment:
https://github.com/cms-sw/cmssw/issues/26385#issuecomment-482567459

Reply:
okay I'll revert this. It was also due to a misunderstanding on units my end, I thought if a single one failed, it would abort things like the comparisions but now I see this is not the case. So this unit test failing is not necessarily a problem for this PR.